images/manager/Dockerfile

```diff
@@ -15,21 +15,21 @@ RUN pip install --upgrade pip && \
 
 RUN apk add --no-cache bash curl gettext jq openssl
 
-RUN curl --location "https://github.com/weaveworks/eksctl/releases/download/0.51.0/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp && \
+RUN curl --location "https://github.com/weaveworks/eksctl/releases/download/v0.67.0/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp && \
```

Comment:
To confirm this, they changed `{major.minor.patch}` to `v{major.minor.patch}`, right?

Reply:
Yes that is correct

Reply:
Got it.